scripts/flow/common.js

```diff
@@ -32,7 +33,7 @@ async function _executeMacroByName(
 	macroName,
 	sourceToken = {},
 	{
-		compendiumName = "lancer-weapon-fx.WeaponFX",
+		compendiumName = `${MODULE_ID}.weaponfx`,
```

Comment:
Small change here: `WeaponFX` -> `weaponfx` for the compendium ID. When making a compendium in-client, it always lowercases the name, so this seems more correct?